docs/fundamentals/toc.yml

```diff
@@ -2544,6 +2544,8 @@ items:
   items:
   - name: Overview
     href: ../core/porting/index.md
+  - name: Port .NET Framework libraries to .NET
+    href: ../core/porting/libraries.md
```

Comment:
@adegeo I assume this wasn't omitted purposefully?

Reply:
Quite possibly. This area is in progress ([project 128](https://github.com/dotnet/docs/projects/128)) and @StephenBonikowsky is handling redesigning and rewriting all of this. It was probably removed when this was restructured and isn't added back in yet until it's rewritten to fit with the migration workflow.

Reply:
@StephenBonikowsky Should I remove [docs/core/porting/libraries.md](https://github.com/dotnet/docs/blob/main/docs/core/porting/libraries.md) from the table of contents? Should the article be deleted and redirected?

Reply:
That file doesn't fit with the new TOC (https://github.com/dotnet/docs/issues/23215).
It's contents will be addressed in numerous other pages most of which don't exist yet.

I would delete it and if it needs to be re-directed maybe just point to the main Overview page (which itself will be getting a significant re-write)
https://docs.microsoft.com/en-us/dotnet/core/porting/